### ISSUE TITLE
Move measurements.* to S3

### ISSUE THREAD

**Issue by matschaffer** [Safecast/safecastapi#571]
Possibly other data exports while we're at it

Our main downloads https://blog.safecast.org/data/ are ~14G and ~3G

![Screen Shot 2019-07-24 at 9.46.14 PM.png](https://images.zenhubusercontent.com/579ea40fe40e5714b16cbfe1/8d9bdb52-5f73-4da4-af0c-5cda9c1dd088)

Even with the increased 10 minute timeout there's no way we could serve them via EC2 in a reasonable time frame.

We should move them to S3, probably into the opendata account so we don't have to pay for the transfer.

We'll want to make sure the old urls work, so we should set up some redirects in the API service (either rails or nginx config is probably fine)

**Comment by matschaffer** [comment]
Probably makes sense to at least shard these by year too so people don't always have to download the whole massive data set.

**Comment by matschaffer** [comment]
I recently learned that tile generation leverages https://api.safecast.org/system/mclean.tar.gz which has a similar limitation.

Would be good to get as many of this large binary objects onto s3 as possible.

**Comment by matschaffer** [comment]
Actually, just tested downloading from s3 and it wasn't much different. Could be a local network problem causing the 1.1G mclean.tar.gz to download so slowly.

**Comment by matschaffer** [comment]
Yeah, I got a gigabit ethernet adapter for the new tile processing mac and the download was way faster.

Would still be good to move these to S3 to avoid unnecessary EC2 transfer, but probably not as pressing as I thought.

**Comment by matschaffer** [comment]
Including the cloudwatch scaleup alert that this ends up triggering as well.

I've switched the app to scale on CPU in the mean time.

![Screen Shot 2020-05-06 at 10 04 43 PM](https://user-images.githubusercontent.com/690/81180179-9c40da00-8fe5-11ea-9324-4885c3916d8d.png)

**Comment by sasharevzin** [comment]
Do we have a testing S3 account keys so I can mess a little bit with it?
